configs/test/batch/batch.yaml

```diff
@@ -67,3 +66,7 @@ mapping:
     preemptible: true
     machine_type: n1-standard-1
 project: 'test-clusterfuzz'
+# TODO(metzman): Come up with a better system than this. I think a system where
+# we have a list of zones and associated config (such as subnets) to pick where
+# to launch tasks is ideal.
+region: 'us-central1'
```

Comment:
what's using this exactly? 

can you also explain a bit why we can't just go with "a system where
# we have a list of zones and associated config (such as subnets) to pick where
# to launch tasks is ideal." in this PR? Are there anything blockign this?

Reply:
It's old. I removed it.
There's nothing blocking it to be honest. But I think it's a bit complex to check multiple regions for CPU usage instead of just one.
I think we can simplify our implementation if we just force all preemptible (fuzz) tasks into us-central1 and all non-preemptible tasks (everything else) into us-east4. We won't need to do anything complicated to pick the region and we will never starve the non-fuzz tasks (and will have great latency for these). The problem with this approach is:
1. Cloud might not appreciate upping our CPU usage by 50k in one region and prefer we split the workload.
2. We may want to fuzz less if we are doing a lot of other tasks, the approach I give above means that our spending is uncapped, it's whatever fuzzing capacity is + how many non-fuzz tasks are requested).